frontend/src/sagas.js

```diff
@@ -2216,6 +2218,22 @@ export function* assignSubprojectSaga({
   }, showLoading);
 }
 
+export function* validationErrorSaga() {
+  yield execute(function*() {
+    yield put({ type: VALIDATION_ERROR_MESSAGE });
+	  yield put({
+      type: SNACKBAR_MESSAGE,
+      message: 'Invalid data'
```

Comment:
`Invalid data` might be misunderstood by the user. The user could think he/she did something wrong. We should show that the error is on the application side and not the users fault, like `Oops!...It's not your fault - A client side validation error occured. Please inform the administrator.`